### DISCUSSION TITLE
Guest Authorization with 2FA Enabled

### DISCUSSION THREAD

**Discussion by majidsikandar** [Art-of-WiFi/UniFi-API-client#230] [Q&A]
guest authorization failed for OS with 2FA enabled.

**Comment by majidsikandar** [comment]
@malle-pietje any thoughts on that?

**Comment by malle-pietje** [comment]
Read this blog post:
https://artofwifi.net/blog/important-update-for-all-unifi-api-users-switch-to-local-admin-accounts-before-july-2024
